Assets/Scripts/Scenes/InGame/Player/PlayerStatus.cs

```diff
@@ -150,6 +158,15 @@ public void ChangeSanValue(int value, string mode)
                 _san.Value = Mathf.Max(0, _san.Value - value);
         }
 
+        /// <summary>
+        /// �ړ����x��ύX������֐�
+        /// </summary>
+        /// <param name="value">�ύX��</param>
+        public void ChangeSpeed(int value)
+        {
+            _speed.Value = value;
+        }
```

Comment:
```suggestion
        private bool _isUseItem = false;
        public void UseItem(bool use)
        {
            _isUseItem = use;
        }

        /// <summary>
        /// 移動速度を変更させる関数
        /// </summary>
        /// <param name="value">変更量</param>
        public void ChangeSpeed()
        {
            _speed.Value =  (int)( _speedBase * (_isUseItem ? 0.5f : 1));
        }
``` 
移動速度の変更ですが、現状だと複数の移動速度変更の効果があると、お互いに干渉してしまいます。
なので、それぞれ、状態をbool値で管理し、ChangeSpeedではその状態によって _speed.Valueを変化させるのはどうですか。
干渉する例.
①影響１で移動速度が半減して2.5になる（影響終了時は_speedBaseの5に戻る）
②影響２で移動速度が半減して1.25になる（影響終了時は_speedBaseの2.5に戻る）
③影響１が終了して移動速度が5に戻る
④影響２が終了して移動速度が2.5に戻る
⑤以降ずっと2.5のままになる。